### ISSUE TITLE
Repeat commands thirds options not work

### ISSUE THREAD

**Issue by AlexAndradeIntrado** [rxhanson/RectanglePro-Community#146]
This option stop working two o three versions

![image](https://user-images.githubusercontent.com/113928106/197534548-8b1e8873-256e-4111-8375-dbeb383080b3.png)

**Comment by rxhanson** [comment]
This works properly for me. Can you add logs and/or a screen recording? Hold option and select view logging in the menu bar menu to see logs.

**Comment by AlexAndradeIntrado** [comment]
[log.txt](https://github.com/rxhanson/RectanglePro-Community/files/9889706/log.txt)

https://user-images.githubusercontent.com/113928106/198672276-c5b8f37e-acbe-4d39-b33c-dcca3779e4da.mov

**Comment by rxhanson** [comment]
Ah, right. Thanks for the recording. So the win arrow keys follow a different flow than all other left/right half actions, and this is by design. I'll have to think this one through, since that would effectively follow a different flow than Windows (which could lead to some confusion that way as well).

**Comment by AlexAndradeIntrado** [comment]
Can you see the attached log.txt?

**Comment by rxhanson** [comment]
Yes. As mentioned above, this is the way it was designed to work (trying to closely mimic Windows). I'll see what I can do in an upcoming release to improve this, though, since it is confusing the way it is now.

**Comment by AlexAndradeIntrado** [comment]
Thanks. That's the most valuable function for me
